### DISCUSSION TITLE
关于知识共享社区的一些想法

### DISCUSSION THREAD

**Discussion by wenwenziy** [openwhu/OpenWHU#54] [General]
最近尝试了很多号称“第二大脑”的个人笔记平台，从试图一网打尽尽可能满足各种收集癖的**印象笔记**，到具有超级强大的看板功能的**Notion**（其实就是一个很简单的设置不同筛选条件的表格，只不过以不同的方式展现出来），再到比较成熟的语雀，外网号称双链笔记开山鼻祖的**roam research**，以及最近对中英文支持更加友好的**wolai**。
先抛开效率、实用性、美观性这些标准不谈，我觉得当下最值得重新拿来思考的议题是：

1. 文字一种承载我们思想的工具，落实到具体的知识、观点，它是否有一个完美的组织方式？
2. 评判一种组织方式或好或坏的标准是什么？
3. 知识共享何以可能？

为什么要从这三个问题来着手考虑？我目前的判断是：我能想到的最完美的作为知识载体的文字组织方式，其最首要的标准是对于一般的社会人（具备基本的尝试、思考能力和词汇储备）来说是否能够完全理解，而同时也不失知识体系内在的逻辑性，看问题视角的专业性。简单来说，就是说人话且有深度。正因此，才有可能在更大的范围内实现社区内的知识共享。

下面抛砖引玉，先对对第一个问题给我的一点思考。首先我们看那些我们使用过的承载知识的那些工具，它们具体是什么样的？最最简陋的，一张纸片，小纸条，随便写点什么，画一些表情啦，符号啦，就给递给另一个人来看。再精致一点，那就是一页一页装订好的书了，个人随手带的笔记本，绘画本，想写什么写什么。但大多数时候，我们会开始注意格式，因为书本不是随便就丢掉的东西，我们还得准备着随时翻看以前记下的东西，所以就有必要写得工整一些，更加清晰易懂。再后来，有了印刷等各种快速制作书籍的手段，我们开始更加注重排版，章节，标题的组织，力图保证读者阅读体验的顺畅，内容的丰富度，逻辑脉络的清晰度等等。可是这种方式在信息时代，有了一个重大的问题，即记录文字或者说“写书”的权利被大大地普及化了，使得无论是没有储备大量知识的普通人，学生，还是专业的研究者，都希望把自己的所思所想记录下来（当然这是非常有意义的一件事）。
这就对一般人的体系化组织能力有了更高的要求。什么叫体系化？这就有点向数学的公理化，把关键的结论分类（默认需要得到承认的公理，需要精炼的定义，以及需要证明的定理），然后就可以更为清晰地把对于一类问题，一类对象的研究所得组织起来。这也很像程序设计的结构、规范，按照它们来办事，可以使得一个项目能够更为方便的维护、更新迭代。因此，在这方面能力不强的人就很容易陷入一种僵局，即收集了一大堆东西，但用的时候仍旧脑袋空空，找起来费劲吃力。在此我目前认为以问题为中心来组织知识的方式比较合理。比如可以用问题-立场-支持-结论的方式来组织人文学科的知识，以问题-解答-支持（证明）的方式来组织理工科的知识，以问题-解决办法-为什么可行来组织需要实践操作的知识。但这种办法的问题在于，（特别是对于人文类学科）看待问题的方式太多太多了，同一个问题完全可以有不同的提法，要做出回答所需考虑的内容也会产生千百种变化，这对于学科素养以及个人内心的直觉判断带来了很大考验。同时也对旁人想要去理解别人的看法带来了很大的困难，所以为什么常常说要看上下文，要了解具体的社会环境，人物背景，再去理解某个历史人物的立场和观点。

待续...

同步到知乎 [链接](https://zhuanlan.zhihu.com/p/386937712)

**Comment by Crescentm** [comment]
待续就不更新了是吧（催更）

**Comment by yqwu905** [comment]
这个似乎和OpenWHU项目的目的不太一样(我对这一项目的理解是我们提供**课程的原始资料和一些自己的见解**, 但知识管理的方法, 乃至于特定的软件, 这个应该是因人而异的)
另外, 谈一些我个人的理解.
1. 没有, 这取决于每个人的认知习惯, 也取决于每个人对知识记录的需求. 例如, 同样对于物理, 有的人更习惯于那种从几个基本假设出发, 然后一口气推到底, 这样像普通书籍那样的**线性**记录方式更适合他, 但有的人更习惯以问题出发, 通过对一个个问题的求解来学习一个学科, 这样的话他可能就更适合其他的组织方式.
2. 我觉得就是对知识所面向群体的平均理解难度与查阅难度.
3. 这对于不同的学科似乎具有不同的难度, 例如对于理工科应当较为简单, 对于医学等文科也较为简单, 但对于文学, 哲学这样的学科来说似乎比较难(我对文科并不是很了解, 如果有错误, 请原谅我)
最后, 目前来看, 知识管理是工具强依赖的, 因此知识分享也是摆脱不了工具的限制, 这就使得分享更加困难, 例如对于理科学生, 对公式的良好支持是很重要的(这也就是我弃了Notion的主要原因), 对于医学/生物学这种更偏集邮式科学观的学科来说, Obsidian这一类提供知识图谱能力的软件也许有很大优势. 因此, 抛开工具来谈似乎并不是很可能(因为绝大多数人都不具备自行去开发一套适合于自己知识管理方法的软件, 而是要去适应最接近自己思维的软件)

**Comment by Pansamic** [comment]
怎么加入计划？
